guides/common/modules/proc_creating-an-installation-medium-for-debian.adoc

```diff
@@ -0,0 +1,23 @@
+[id="Creating_an_Installation_Medium_for_Debian_{context}"]
+= Creating an Installation Medium for Debian
+
+Create an installation medium in {Project} to provision hosts running Debian or Ubuntu.
+
+ifdef::orcharhino[]
+[TIP]
+====
+You can use an Ansible role to configure operating systems to skip this step.
+The Ansible role is located on your orcharhino Server at `/usr/share/orcharhino-ansible/roles/or_operating_systems/`.
+====
+endif::[]
```

Comment:
Oh I want this in Foreman core someday too :-)

Reply:
It would probably need to be made quite a bit more generic, to add it to the upstream collections.
The next role, I do want to add to the collection is the one to configure Katello to sign deb content.